.github/ISSUE_TEMPLATE/3_discussion.yml

```
@@ -0,0 +1,7 @@
+name: 💬 Ask a question or open a discussion
+description: Issues that don't have a specific topic
+labels: under discussion
+body:
+  - type: textarea
+    attributes:
+      label: Ask a question or open a discussion
```

Is this required? If so
```suggestion
      label: What’s up?
```

😃

---

Yes, it's required.